### ISSUE TITLE
如何获得简体中文插件包的历史版本？

### ISSUE THREAD

**Issue by IronAspen** [microsoft/vscode-loc#1206]
当前代码开发基于WIN7平台，最高支持1.70.3版本，因此无法安装最新版本简体中文包。
插件市场历史版本仅显示最近版本，无之前版本下载链接，请问是否有下载方法？

**Comment by v-mholloway** [comment]
@IronAspen @NatureLR Can you please supply your feedback in English to have the issue processed?

**Comment by IronAspen** [comment]
> 插件里选择安装另一个版本 选个比较早点的就能恢复中文了 ![image](https://user-images.githubusercontent.com/30221928/206346266-08363081-c232-4176-9c72-92dcac8b0fdf.png)

抱歉我没有描述清楚我的问题。因为我需要离线安装插件，因此需要下载 .vsix版本的离线安装包，但是在官方插件市场仅有最新版本的安装包提供下载，尝试了通过API下载的方法但是无法找到对应版本的URL。目前需要下载1.70.2版本可用的.vsix安装包。非常感谢您的回答！

**Comment by IronAspen** [comment]
> @IronAspen @NatureLR Can you please supply your feedback in English to have the issue processed?

Of course, I want to download the .vsix package which is avaliable for 1.70.2 VsCode to offline install simplified Chinese language pack, but i don't know how can I download it. only the most recent version can be found in the market.

**Comment by Hisioni** [comment]
No more early historical version found on the official website
